charts/latest/azurefile-csi-driver/templates/csi-azurefile-controller.yaml

```diff
@@ -193,6 +193,16 @@ spec:
             timeoutSeconds: 10
             periodSeconds: 30
           env:
+{{- if and .Values.workloadIdentity.clientID .Values.workloadIdentity.tenantID}}
+            - name: AZURE_CLIENT_ID
+              value: {{ .Values.workloadIdentity.clientID }}
+            - name: AZURE_TENANT_ID
+              value: {{ .Values.workloadIdentity.tenantID }}
+            - name: AZURE_FEDERATED_TOKEN_FILE
+              value: /var/run/secrets/tokens/azure-identity-token
+            - name: AZURE_AUTHORITY_HOST
+              value: https://login.microsoftonline.com/
```

Comment:
this is the global setting, what if not set? and also need to support sovereign cloud.